crates/ra_assists/src/handlers/fill_match_arms.rs

```diff
@@ -2,6 +2,8 @@
 
 use std::iter;
 
+use itertools::Itertools;
+
```

Comment:
We generally treat all non-std external crates the same, so there's no need for a blank line here.

Reply:
Fixed.